### PR TITLE
handle more methods

### PR DESCRIPTION
**Pull request by yanns**

### CODE REVIEW

File: src/backend.rs [hunk 1]
```diff
@@ -57,14 +57,29 @@ impl Serialize for HttpSerializer {
...
-              Host: www.example.com\r\n\
-              Accept: */*\r\n\
-              \r\n";
-
-        trace!("Trying to write {} bytes", input.len());
-        buf.copy_from_slice(&input[..]);
-        trace!("Copied {} bytes", input.len());
+        match msg {
+            Frame::Message { message, body: _ } => {
+                let input = format!(
```
Comment (hjr3) [inline src/backend.rs]: Nice. I created https://github.com/hjr3/alacrity/issues/15 to fix my hack too.

**Comment by hjr3** [comment]
Travis is having problems again. I am confident the tests are passing on stable since they are passing on nightly and beta. I also tested it locally.

**Comment by hjr3** [comment]
@yanns the merging of Pool Management API PR caused a merge conflict. I cannot update your branch directly, but the fix is dead simple. I can PR it if you want as well.

**Comment by yanns** [comment]
I'll rebase the commit. Thx for proposing!

**Comment by yanns** [comment]
rebase done.

**Pull request merged**